index.bs

```diff
@@ -546,98 +546,57 @@ Modify {{Fence/reportEvent()}} as follows:
 
 <div algorithm="report-event-monkeypatch">
 
-Replace the step
-
-> Run <a spec=fenced-frame>report an event</a> using |instance|'s
-> [=fenced frame config instance/fenced frame reporter=] with |destination|,
-> |event|'s {{FenceEvent/eventType}}, and |event|'s {{FenceEvent/eventData}}.
-
-with
-
-Run <a spec=fenced-frame>report an event</a> using |instance|'s
-[=fenced frame config instance/fenced frame reporter=] with |destination|,
-|event|'s {{FenceEvent/eventType}}, |event|'s {{FenceEvent/eventData}},
-and [=this=]'s [=relevant global object=]'s [=associated Document=].
-
-</div>
-
-Modify <a spec=fenced-frame>report an event</a> as follows:
-
-<div algorithm="report-an-event-monkeypatch">
+Before the step
 
-Add a [=Document=] parameter called |document|.
-
-After the step
-
-> If |reporting map|[|destination|] does not [=map/exist=], return.
+> [=list/For each=] |destination| of <var ignore=''>event</var>'s {{FenceEvent/destination}}:
```

Comment:
In what algorithm? You should probably link to it in the "Before the step in the [....] algorithm" above, to make this much more clear. Or maybe I'm missing something? Basically I wouldn't reference an unattached step like this...

Reply:
Thanks, it's in the reportEvent() algorithm (please expand the unchanged text above), but added a few more steps to be clearer.

Reply:
Ah, sorry I missed that.